WordPress/Classes/Services/ReaderSiteService.m

```diff
@@ -204,6 +203,21 @@ - (void)flagSiteWithID:(NSNumber *)siteID asBlocked:(BOOL)blocked success:(void(
     }];
 }
 
+- (void)topicWithSiteURL:(NSURL *)siteURL success:(void (^)(ReaderSiteTopic *topic))success failure:(void(^)(NSError *error))failure
+{
+    WordPressComRestApi *api = [self apiForRequest];
+    ReaderSiteServiceRemote *service = [[ReaderSiteServiceRemote alloc] initWithWordPressComRestApi:api];
+    
+    [service findSiteIDForURL:siteURL success:^(NSUInteger siteID) {
+        ReaderTopicService *topicService = [[ReaderTopicService alloc] initWithManagedObjectContext:self.managedObjectContext];
+        NSNumber *site = [[NSNumber alloc] initWithUnsignedLong:siteID];
```

Comment:
Purely a matter of stylistic choice, there's `[NSNumber numberWithUnsignedLong:]` if you prefer to not use `alloc`.

Reply:
Oh thanks! alloc init is pure habit with ObjC. 😄